app/core/templates/_saved_topics_list.html

```
@@ -0,0 +1,74 @@
+{% if topics %}
+{% if show_controls %}
+<div class="topics-controls-toolbar">
+    <div class="topics-search-wrapper">
+        <input type="text" id="topic-search" placeholder="🔍 Search topics..." autocomplete="off">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)